translates: hooks-state (#68)
reactjs/nl.reactjs.org: content/docs/hooks-state.md

---
id: hooks-state
title: De State Hook
permalink: docs/hooks-state.html
next: hooks-effect.html
prev: hooks-overview.html
---

*Hooks* zijn een nieuwe toevoeging in React 16.8. Ze maken het mogelijk om state en andere React voorzieningen te gebruiken zonder dat je een class hoeft te schrijven.

De [introductiepagina](/docs/hooks-intro.html) gebruikte dit voorbeeld om vertrouwd te raken met Hooks:

```js{4-5}
import React, { useState } from 'react';

function Example() {

>   // Declareer een nieuwe state variabele, de we "count" zullen noemen

Reviewer: ```suggestion
  // Declareer een nieuwe state variabele, die we "count" zullen noemen
```

  render() {
    return (
      <div>
        <p>Je klikte {this.state.count} keer</p>
        <button onClick={() => this.setState({ count: this.state.count + 1 })}>
          Klik me
        </button>
      </div>
    );
  }
}
```

> De state begint als `{ count: 0 }`, en we verhogen `state.count` als de gebruiker op een knop druk door `this.setState()` aan te roepen. We zullen stukjes van deze class gebruiken over de hele pagina verspreid.

Reviewer: ```suggestion
De state begint als `{ count: 0 }`, en we verhogen `state.count` als de gebruiker op een knop druk door `this.setState()` aan te roepen. We zullen stukjes van deze class her en der in de pagina gebruiken.
```
